packages/theme-service/src/hooks/useThemeLayoutEffect.spec.ts

```diff
@@ -9,16 +9,41 @@ describe('useThemeLayoutEffect', () => {
   it('should add a style element to the head and add the light class to the body', () => {
     const themeName = 'light' as ThemeName
 
-    renderHook(() => useThemeLayoutEffect(themeName, lightColors, darkColors))
+    const { rerender } = renderHook(() => useThemeLayoutEffect(themeName, lightColors, darkColors))
+
+    expect(document.head.innerHTML).toContain('body.light')
+    expect(document.body.classList).toContain('light')
+
+    rerender({ themeName, lightColors, darkColors })
+
     expect(document.head.innerHTML).toContain('body.light')
     expect(document.body.classList).toContain('light')
+
+    rerender({ themeName: 'dark', lightColors, darkColors })
+    expect(document.head.innerHTML).toContain('body.dark')
```

Comment:
need update props on rerender, check adding both styles + check that body class name changes on rerender with new theme name

```
    const { rerender } = renderHook(
      ([themeName, lightColors, darkColors]) => useThemeLayoutEffect(themeName, lightColors, darkColors),
      { initialProps: ['light', lightColors, darkColors] },
    )

    expect(document.head.innerHTML).toContain('body.light')
    expect(document.head.innerHTML).toContain('body.dark')
    expect(document.body.classList).toContain('light')
    expect(document.body.classList).not.toContain('dark')

    rerender(['light', lightColors, darkColors])

    expect(document.body.classList).toContain('light')
    expect(document.body.classList).not.toContain('dark')

    rerender(['dark', lightColors, darkColors])
    expect(document.body.classList).toContain('dark')
    expect(document.body.classList).not.toContain('light')
 ```